flow/layers/backdrop_filter_layer_unittests.cc

```diff
@@ -0,0 +1,188 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/flow/layers/backdrop_filter_layer.h"
+
+#include "flutter/flow/testing/layer_test.h"
+#include "flutter/flow/testing/mock_layer.h"
+#include "flutter/fml/macros.h"
+#include "flutter/testing/mock_canvas.h"
+#include "third_party/skia/include/core/SkImageFilter.h"
+#include "third_party/skia/include/effects/SkImageFilters.h"
+
+namespace flutter {
+namespace testing {
+
+static constexpr SkRect kEmptyRect = SkRect::MakeEmpty();
+
+using BackdropFilterLayerDeathTest = LayerTest;
```

Comment:
Here and elsewhere: Why a separate fixture for death and non-death tests? You are not doing anything differently. If you wanted to though, maybe switching to threadsafe death assertions might be a way to go.

Reply:
Yeah, I didn't know about the threadsafe assertions.  Gonna switch to those